### ISSUE TITLE
Select columns or hide columns?

### ISSUE THREAD

**Issue by Yevgnen** [YS-L/csvlens#12]
When there are many columns, it would be nice to have ability to select or hide only some columns.

**Comment by ondohotola** [comment]
Use [QSV](https://github.com/jqnatividad/qsv) to select fewer columns.

**Comment by michaelmior** [comment]
I agree this would be very helpful. Of course it's possible to remove some columns before running in csvlens, but sometimes I want to hide and show columns dynamically in the same session.
